internal/builders/generic/attest.go

```diff
@@ -180,3 +193,9 @@ run in the context of a Github Actions workflow.`,
 
 	return c
 }
+
+// isPresubmitTests returns true if running in pre-submit tests.
+func isPreSubmitTests() bool {
```

Comment:
shall we move this into the library? (It's going to be duplicated in all builders)

Reply:
I'd rather that it not be a public API so maybe it can go under `internal/` somewhere.